.github/workflows/percy.yml

```diff
@@ -1,23 +1,15 @@
-name: Tests
+name: Percy
 on:
   push:
     paths:
       - 'js/**'
       - 'scss/**'
-      - 'site/content/docs/**/components'
-      - 'site/content/docs/**/forms'
+      - 'site/content/docs/**/components/**'
+      - 'site/content/docs/**/forms/**'
     branches:
       - main
-      - "!dependabot/**"
+      - v4-dev
   pull_request:
```

Comment:
The goal of this PR is to run both jobs when there is a push on _main_ or _v4-dev_ or on demand manually. It is needed to remove this line isn't it?